dataactbroker/handlers/fileHandler.py

```diff
@@ -782,9 +782,13 @@ def check_detached_generation(job_id):
                          'start': '', 'end': ''}
 
         if upload_job is None:
-            response_dict['status'] = 'invalid'
+            response_dict['status'] = 'Invalid'
             response_dict['message'] = 'No generation job found with the specified ID'
             return JsonResponse.create(StatusCode.OK, response_dict)
+        elif upload_job.filename is None:
+            response_dict['status'] = 'Invalid'
```

Comment:
Is frontend doing a case-insensitive check on the status? Just asking since the it was capitalized and wanted to make sure there were no issues with that when using the response